include/maths/analytics/CBoostedTreeFactory.h

```diff
@@ -387,6 +394,7 @@ class MATHS_ANALYTICS_EXPORT CBoostedTreeFactory final {
     TBoostedTreeImplUPtr m_TreeImpl;
     mutable std::size_t m_PaddedExtraColumns{0};
     TTrainingStateCallback m_RecordTrainingState{noopRecordTrainingState};
+    THyperparametersDoublePrVecSPtr m_HyperparametersLosses;
```

Comment:
This feels misplaced to me here. They belong in `CBoostedTreeHyperparameters` IMO since it is responsible for maintaining all other state related to hyperparameter optimisation.

Reply:
Thank you for the suggestion. I moved the member variable into `CBoostedTreeHyperparameters` :+1: